lib/ve_logger_filter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = (hyper, req, next, options, specInfo) => {
+
+    options.probability = options.probability || 1;
+
+    if (req.headers && req.headers['user-agent'] &&
+            req.headers['user-agent'].includes('VisualEditor') &&
+            Math.floor(1000 * Math.random()) <= Math.floor(1000 * options.probability)) {
+        hyper.logger.log('warn/visualeditor', {
+            msg: 'VisualEditor request',
+            req_headers: req.headers
```

Comment:
Let's json-stringify them.. Could be too much cardinality for logstash to handle